### DISCUSSION TITLE
Idea: add ability to install cia's from nds files.

### DISCUSSION THREAD

**Discussion by CarrotFarmering** [d0k3/GodMode9#680] [Ideas]
So, i have some Ds games on my sd card, but i cant install cia's from them. I tried installing the 1.9.3pre1 version, since it has dsi compatibility, but nope. Still can't.

**Comment by d0k3** [comment]
You're mixing something up here. The 3DS can run DSiWare natively, as it was intended, from installed memory.

DS carts and DS cart dumps are not the same as DSiWare, and the 3DS can only run such titles from the cart slot natively. I could build a CIA from any kind of file actually, but it won't run. Try TwiLightMenu instead, it has become pretty good.

**Comment by awes145-neo** [comment]
You could try forwarders.
sthetix Has a guide.
